components/raftstore/src/store/fsm/apply.rs

```diff
@@ -4068,10 +4068,8 @@ where
 
         let mut entries = Vec::new();
 
-        let mut dangle_size = 0;
         for cached_entries in apply.entries {
-            let (e, sz) = cached_entries.take_entries();
-            dangle_size += sz;
+            let (e, _) = cached_entries.take_entries();
```

Comment:
If no cache eviction occurs, we still need to account for the size.

Reply:
@hhwyt Sorry, I don't understand why we need to record the size of entry in Apply::Task by `RAFT_ENTRIES_CACHES_GAUGE`